Source/Text/SvgTextBase.cs

```diff
@@ -400,7 +400,8 @@ private void SetPath(TextDrawingState state, bool doMeasurements)
                         {
                             if (this.X.Count < 2)
                             {
-                                origState.LetterSpacingAdjust = -1 * diff / (state.NumChars - origState.NumChars - 1);
+                                if (state.NumChars - origState.NumChars - 1 != 0)
+                                    origState.LetterSpacingAdjust = -1 * diff / (state.NumChars - origState.NumChars - 1);
                                 SetPath(origState, false);

```

Comment:
@mrbean-bremen 

I think that may be the following code because it does not change `state`.
```cs
if (state.NumChars - origState.NumChars - 1 != 0)
{
    origState.LetterSpacingAdjust = -1 * diff / (state.NumChars - origState.NumChars - 1);
    SetPath(origState, false);
    return;
}
```

Reply:
Yes, this is the point I wasn't sure about - I don't really understand that code either... I will have a closer look later, thanks!

Reply:
Ok, I agree with you - I changed it as you proposed..

Reply:
@H1Gdev: On a general note - I'm not sure if I will do much for SVG.NET in the next time (I'm a bit short on time lately), but I would appreciate if you could review my stuff if you get the time. I generally prefer a second pair of eyes - @tebjan doesn't really have the time, and you seem to know quite a bit of the code already, so that would be really helpful.